python/doc/theory/meta_modeling/cross_validation.rst

```diff
@@ -1,186 +1,654 @@
 .. _cross_validation:
 
-Cross validation assessment of PC models
-----------------------------------------
-
-Once a polynomial response surface :math:`\widehat{h}(\underline{x})`
-of the original numerical model :math:`h(\underline{x})` has been
-built up, it is crucial to estimate the approximation error, i.e. the
+Validation and cross validation of surrogate models
+---------------------------------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+In ordinary validation of a surrogate model, we hold some observations from the
+sample and train the model on the remaining observations.
+Then we test the surrogate model against the hold out observations.
+One of the problems of this method is that the surrogate
+model is trained using only a subset of the available data.
+As a consequence, the estimated error can be pessimistic.
+In cross-validation, the roles of the train and test samples are exchanged.
+Hence, more of the available data is used, resulting in an improved
+accuracy of the estimated error of the metamodel.
```

Comment:
You are right: alternating between both words is confusing.

But "surrogate model" cannot be specific to the French community... because it is not French! So, let's benchmark this.
- "Metamodel": 14 languages
- "Surrogate model": 3 languages

So "Metamodel" wins.

![image](https://github.com/openturns/openturns/assets/31351465/adc9af62-37ce-44d0-8ce0-846e5712ecd3)

![image](https://github.com/openturns/openturns/assets/31351465/4d522760-d2d2-49b1-bb20-98ff80c7c450)

Reply:
Fixed at a7221cdfd8c7e9842c53ef8bfbd6f5d70791831a